testdata/curlget.go

```diff
@@ -5,13 +5,16 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"strings"
 )
 
 func main() {
 	if len(os.Args) != 2 {
 		log.Fatal("Usage: go run curlget.go <url>")
```

Comment:
The Usage should mention that `$TASKCLUSTER_PROXY_URL` is (or env vars of the form `${`_env_var_`}` are) expanded.